### PR TITLE
add browser/firefox tag for plugins and an extra app match

### PR DESCRIPTION
**Pull request by fidgetingbits**
also select all which is nice for the url bar

### CODE REVIEW

File: apps/linux/firefox.talon [hunk 1]
```diff
@@ -1,9 +1,9 @@
 os: linux
 app: Firefox
+app: firefox
 -
-#todo tags
-# settings():
-	# tags = browser
+tag(): browser
+tag(): firefox
```
Comment (knausj85) [inline apps/linux/firefox.talon:6]: I think this needs to be `user.firefox`, no?

File: apps/linux/firefox.talon [hunk 2]
```diff
@@ -72,3 +72,5 @@ action(browser.submit_form):
 
 action(browser.toggle_dev_tools):
 	key(ctrl-shift-i)
+
+select all: key(ctrl-a)
```
Comment (knausj85) [inline apps/linux/firefox.talon:76]: I can't believe I missed this command. There's an existing action that's implemented in edit.talon, `edit.select_all`

I think 'select all' should be moved here - 
https://github.com/knausj85/knausj_talon/blob/master/text/generic_editor.talon

File: apps/linux/firefox.talon (hunk 1 above)
Comment (fidgetingbits) [reply apps/linux/firefox.talon:6]: not for tags afaik. my vimvixen talon file triggers based on `tag: firefox` with this np.

i agree it's kinda weird though, as it differs from modes, settings, etc namespace

File: apps/linux/firefox.talon (hunk 2 above)
Comment (fidgetingbits) [reply apps/linux/firefox.talon]: sounds good, moved it and just pushed it for this PR

**Pull request merged**